comments-bundle/contao/dca/tl_module.php

```diff
@@ -69,3 +70,32 @@
 	'eval'                    => array('includeBlankOption'=>true, 'chosen'=>true, 'tl_class'=>'w50'),
 	'sql'                     => "varchar(64) COLLATE ascii_bin NOT NULL default ''"
 );
+
+$bundles = System::getContainer()->getParameter('kernel.bundles');
+
+if (isset($bundles['ContaoNewsBundle']))
```

Comment:
I would prefer to add this to a/the listener as well, the DCA code should not contain conditional stuff (e.g. something that would not be possible in YAML) imho. I know that's not always the case atm, but kinda best practice 😅